apps/docs/pages/guides/database/inspect.mdx

```diff
@@ -0,0 +1,174 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'json',
+  title: 'Inspecting your Postgres database',
+  description:
+    'Inspecting your Postgres database for common issues around disk, query performance, index, locks, and more using the terminal.',
+}
+
+## Inspect from the terminal
```

Comment:
it's good to start with a paragraph rather than a title - I think we can remove this one and it will give a better flow from the main page title